framework/src/task_manager/task_manager_core.c

```diff
@@ -317,7 +332,13 @@ static int taskmgr_stop(int handle, int caller_pid)
 	/* Call stop callback */
 	TM_STATUS(handle) = TM_APP_STATE_CANCELLING;
 	if (TM_STOP_CB_INFO(handle) != NULL) {
-		(*TM_STOP_CB_INFO(handle)->cb)((void *)TM_STOP_CB_INFO(handle)->cb_data);
+		if (STOP_CBDATA(handle) != NULL) {
```

Comment:
Don't you need to check CBFUNC for null? No problem with CBDATA null. But CBFUNC can cause crash.